settings.php

```diff
@@ -18,33 +18,32 @@
  * along with this library. If not, see <http://www.gnu.org/licenses/>.
  */
 
-OC_Util::checkAdminUser();
+OCP\User::checkAdminUser();
 OCP\Util::addStyle(APP_NAME, 'settings');
 OCP\Util::addScript(APP_NAME, 'settings');
 
 $params = array('identity_provider', 'mail_attr', 'persistent_id_attr', 'full_name_attr', 'sessions_handler_url', 'session_initiator_location', 'federation_name', 'enforce_domain_similarity', 'link_to_ldap_backend', 'ldap_link_attribute', 'ldap_uuid_attribute', 'external_user_quota');
-  
+
 if($_POST) {
 	foreach($params as $param) {
 		if (isset($_POST[$param])) {
-			OCP\Config::setAppValue(APP_NAME, $param, $_POST[$param]);
+			\OC::$server->getConfig()->setAppValue(APP_NAME, $param, $_POST[$param]);
 		}
 	}
 }
 
 // fill template
 $tmpl = new OCP\Template( 'user_shibboleth', 'settings');
-$tmpl->assign('sessions_handler_url', OCP\Config::getAppValue('user_shibboleth', 'sessions_handler_url', '/Shibboleth.sso'));
-$tmpl->assign('session_initiator_location', OCP\Config::getAppValue('user_shibboleth', 'session_initiator_location', '/Login'));
-$tmpl->assign('federation_name', OCP\Config::getAppValue('user_shibboleth', 'federation_name', ''));
-$tmpl->assign('enforce_domain_similarity', OCP\Config::getAppValue('user_shibboleth', 'enforce_domain_similarity', '1'));
-$tmpl->assign('link_to_ldap_backend', OCP\Config::getAppValue('user_shibboleth', 'link_to_ldap_backend', '0'));
-$tmpl->assign('ldap_link_attribute', OCP\Config::getAppValue('user_shibboleth', 'ldap_link_attribute', 'mail'));
-$tmpl->assign('ldap_uuid_attribute', OCP\Config::getAppValue('user_shibboleth', 'ldap_uuid_attribute', 'dn'));
-$tmpl->assign('external_user_quota', OCP\Config::getAppValue('user_shibboleth', 'external_user_quota', ''));
-$tmpl->assign('identity_provider', OCP\Config::getAppValue('user_shibboleth', 'identity_provider', 'Shib-Identity-Provider'));
-$tmpl->assign('mail_attr', OCP\Config::getAppValue('user_shibboleth', 'mail_attr', 'mail'));
-$tmpl->assign('persistent_id_attr', OCP\Config::getAppValue('user_shibboleth', 'persistent_id_attr', 'eppn'));
-$tmpl->assign('full_name_attr', OCP\Config::getAppValue('user_shibboleth', 'full_name_attr', 'cn'));
-
+$tmpl->assign('sessions_handler_url', \OC::$server->getConfig()->getAppValue('user_shibboleth', 'sessions_handler_url', '/Shibboleth.sso'));
```

Comment:
Assign \OC::$server->getConfig() to a variable and use it for subsequent lookups.